### DISCUSSION TITLE
How do I fix window shadow on 11.0.0-preview5?

### DISCUSSION THREAD

**Discussion by maguro1kan** [AvaloniaUI/Avalonia#10561] [General]
Hi. How do I fix window shadow on 11.0.0-preview5?

![image](https://user-images.githubusercontent.com/30754124/223014865-4cd5d6ce-77d1-47d3-918d-73d6466eef18.png)

- ExtendClientAreaToDecorationsHint="True"
- ExtendClientAreaChromeHints="NoChrome"
- ExtendClientAreaTitleBarHeightHint="-1"

Thanks

**Comment by timunie** [comment]
Can you double-check if it works in latest nightly build for you? At least for me it works: 

![image](https://user-images.githubusercontent.com/47110241/223045971-9334726d-092f-4305-a7ba-7a25827b01ec.png)

**Reply by timunie** [reply]
OS: Windows 11
